src/modules/projectlist.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-undef */
+/*  eslint-disable  no-unused-vars */
+
+export const projectList = [];
+export const globalTaskId = [];
+export function deletePeojectFromArray(arr, value) {
+  let i = 0;
+  while (i < arr.length) {
+    if (arr[i].ProjectId === value) {
+      arr.splice(i, 1);
+    } else {
+      ++i;
+    }
+  }
+  return arr;
+}
+
+export function getPtojectId(element) {
+  return target.parentNode.id;
+}
```

Comment:
[optional] define the functions normally, then great and export object with just the name of each function, cleaner code :slightly_smiling_face: